app/src/main/java/org/mozilla/focus/download/DownloadInfo.java

```diff
@@ -11,22 +14,27 @@
 
 public class DownloadInfo {
 
-    public static final String STATUS_PAUSED = "PAUSE";
-    public static final String STATUS_PENDING = "PENDING";
-    public static final String STATUS_RUNNING = "RUNNING";
-    public static final String STATUS_SUCCESSFUL = "SUCCESSFUL";
-    public static final String STATUS_FAILED = "FAILED";
-    public static final String STATUS_UNKNOWN ="UNKNOWN";
-
+    private Context mContext;
     private Long DownloadId;
-    private String Status;
+    private String StatusStr;
+    private int StatusInt;
     private String Size;
     private String Date;
     private String FileName;
     private String MediaUri;
     private String MimeType;
     private String FileUri;
 
+    public DownloadInfo(Context context){
```

Comment:
Data object shouldn't reference to Context, keep only the status code and translate to string when doing data-view binding.